src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java

```diff
@@ -36,9 +36,6 @@
 import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.servlet.*;
 import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.websocket.AddPathParamQuickFix;
 import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.websocket.WebSocketConstants;
-import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveAbstractModifierQuickFix;
-import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveFinalModifierQuickFix;
-import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveStaticModifierQuickFix;
```

Comment:
JakartaCodeActionHandler is not being used anymore. It's not necessary to make changes to it.

Reply:
@dessina-devasia Please revert this file changes.

Reply:
Reverted changes in JakartaCodeActionHandler